app/helpers/orchestration_stack_helper/textual_summary.rb

```diff
@@ -12,7 +12,7 @@ def textual_group_lifecycle
   end
 
   def textual_group_relationships
-    %i(ems_cloud service orchestration_template instances security_groups cloud_networks parameters outputs resources)
+    %i(ems_cloud service parent_orchestration_stack child_orchestration_stack orchestration_template instances security_groups cloud_networks parameters outputs resources)
```

Comment:
In general a stack can have multiple `child_orchestration_stack`. How are you going to present?